File: dongmin/week7/없는 숫자 더하기.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        List<Integer> num = new ArrayList<>();
        for(int i:numbers)
            num.add(i);     //numbers의 값들을 List형 num에 넣어줌
        for (int i = 0; i < 10; i++) {
            if (!num.contains(i))       //없는 값을 answer에 더함
                answer += i;
        }

        return answer;
    }
}

File: jeonghyeon/7week/없는 숫자 더하기.js
function solution(numbers) {
    let sum = 0;
    for(let i = 0; i <= 9; i++) {
        if(!numbers.includes(i)) { // numbers배열에 0~9까지의 숫자가 있는지 확인하고 없으면
            sum += i; // 더한다
        }
    }
    return sum; // 없는 수들 합해서 반환
}